assets/src/components/PopupMessage.js

```diff
@@ -17,36 +18,34 @@ function PopupMessage ({ a, assignmentGroups }) {
   const dropHighest = getAssignmentRules(a, assignmentGroups).dropHighest
   const dropLowest = getAssignmentRules(a, assignmentGroups).dropLowest
 
+  let rulesMessage
   if (dropHighest !== 0 && dropLowest !== 0) {
-    return (
-      <Typography>
-        {
-          `The highest ${dropHighest} scores will be dropped from this assignment group
-              The lowest ${dropLowest} scores will be dropped from this assignment group`
-        }
-      </Typography>
-    )
+    rulesMessage = `Both the highest ${dropHighest} scores and lowest ${dropLowest} scores will be dropped from this assignment group`
```

Comment:
Just a thought, this line is exceeding max character limit that IDE suggests, standard doesn't seem to see this a problem. I wonder we need to add any custom rules. But not relevant to this PR.

Reply:
I fixed this using some re-formatting. Yeah, not sure if `standard` has a line length rule...